Source/ThirdParty/libwebrtc/Source/webrtc/sdk/objc/components/video_codec/nalu_rewriter.cc

```diff
@@ -678,7 +678,7 @@ static uint8_t ComputeH264ReorderSizeFromSPS(const SpsAndVuiParser::State& state
   }
 
   uint64_t max_dpb_mbs = maxDpbMbsFromLevelNumber(state.profile_idc, state.level_idc, state.constraint_set3_flag);
-  uint64_t max_dpb_frames_from_sps = max_dpb_mbs / ((state.width / 16) * (state.height / 16));
+  uint64_t max_dpb_frames_from_sps = max_dpb_mbs / ((state.pic_width_in_mbs_minus1 + 1) * (state.pic_height_in_map_units_minus1 + 1));
```

Comment:
it's suspicious as before height was calculated as `16 * (2 - sps.frame_mbs_only_flag) * (pic_height_in_map_units_minus1 + 1);` we lose this conversion now.
and the width could never be 0 unless no SPS was encountered.